### PR TITLE
Update docs and examples about local setup

### PR DESCRIPTION
**Pull request by m1n0**
Some small updates to docs.

### CODE REVIEW

**Review by paveldedik — CHANGES_REQUESTED** [review]
Thank you for the fixes! Please, check the comments :-)

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ZOO_DEBUG=1
```
[the remark this answers is not in the document]
Comment (aexvir) [reply .env.example]: > I would consider removing https://github.com/kiwicom/the-zoo/blob/master/docker-compose.dev.yml in favour of this .env.example

Fully agree, I think we should definitely get rid of the `docker-compose.dev.yml` like we did in our internal repo. We never used `docker-compose` to deploy this, and it's always annoying to maintain both files.

[the remark this answers is not in the document]
Comment (m1n0) [reply .env.example]: Issue created here https://github.com/kiwicom/the-zoo/issues/216, file removed for now.

File: README.md [hunk 2]
```diff
@@ -29,6 +29,16 @@ The Zoo also provides analytics about how dependency usage and its versions evol
 
 ## Development
 
+### Initial setup
+
+- Create a database: `$ make migrate`
+- Create a superuser: `$ make superuser`
+- Create `.env` file based on `.env.example`
```
[the remark this answers is not in the document]
Comment (m1n0) [reply README.md]: Done

**Review by m1n0 — COMMENTED** [review]
Updated and created an issue.

File: README.md [hunk 3]
```diff
@@ -56,6 +59,20 @@ Access web locally:
 Check `Makefile` for shell commands if you want to run them with modified
 parameters.
 
+### Local .env file
+
+Local `.env` file is required to run the application, a basic example of such file contains the following:
```
Comment (paveldedik) [inline README.md]: Technically, some component's don't require any configuration. So I would change it like this (just apply the suggestion if you agree)...
```suggestion
Creating a local `.env` file (see [Docker docs](https://docs.docker.com/compose/env-file/)
on `.env` files) is necessary for some components in the app to work correctly,
a basic example of such file contains the following:
```

File: README.md [hunk 4]
```diff
@@ -56,6 +59,22 @@ Access web locally:
...
+```
+ZOO_DEBUG=1
+ZOO_GITHUB_TOKEN=...
+ZOO_GITLAB_TOKEN=...
+ZOO_GITLAB_URL=https://gitlab.com
+```
+
+Version control tokens only need read access to repositories. 
+
+
```
Comment (paveldedik) [inline README.md]: Please remove this blank line, coala is failing because of that.

**Review by paveldedik — COMMENTED** [review]
Looks good! Just fix the coala issue, and you can also squash the commits into one while doing that :-)

File: README.md (hunk 4 above)
Comment (m1n0) [reply README.md]: done

**Review by paveldedik — CHANGES_REQUESTED** [review]
Please squash the commits into one and fix the `Shortlog of the HEAD commit contains 87 character(s). This is 15 character(s) longer than the limit (87 > 72)` issue reported by coala.

**Comment by m1n0** [comment]
Updated and squashed

**Review by paveldedik — APPROVED** [review]

**Comment by paveldedik** [comment]
Thank you!

**Pull request merged**